### DISCUSSION TITLE
DaisyUI + Radix React

### DISCUSSION THREAD

**Discussion by vnugent** [saadeghi/daisyui#1018] [Q&A]
Wow, first off thank you for creating such a beautiful and well-thought out library!

Our project is NextJS + TailwindCSS.  It's been working well for us during prototyping stage.  We are now looking for a component library so that we don't write our own UI components.

One thing I'm not 100% clear is how to use Daisy in React.  For example, if I want to use Daisy [Bottom nav](https://daisyui.com/components/bottom-navigation/) I still have to manage its active state in React.  Something I'm trying to avoid.  Which leads me to think, maybe I can combine Daisy with a headless ui React framework such as [@HeadlessUI](https://headlessui.com/) or [Radix primitives](https://www.radix-ui.com/docs/primitives/overview/introduction)?

In this particular example, I'd use Daisy bottom nav classes to style  Radix [Tabs](https://www.radix-ui.com/docs/primitives/components/tabs).  Is there any reason why combing Daisy with a headless React UI framework is a bad idea?  I did some google search but it seems no one is doing this.

**Comment by wuarmin** [comment]
DaisyUi is just a design. Handling GUI state in react is totally valid and it's the main purpose of react. Using a headless library to manage a GUI component is great too. I.e. I use react-table and use DaisyUi's table component. It's great!

**Reply by vnugent** [reply]
I wanted to make sure I'm not missing something obvious given there's also a new [React Daisy](daisyui.com/?path=/story/welcome--page) project.  IMO combining Daisy with a headless React UI library is a huge win.

**Reply by wuarmin** [reply]
Oh, I didn't know about the new daisyUi react integration. I am also on react and I will stay with using DaisyUi directly. I haven't had such good experiences with such integrations in the past. maybe it's different in this case, but the fewer dependencies, the better.

**Comment by benjitrosch** [comment]
@vnugent You could always try https://github.com/daisyui/react-daisyui 😄

**Reply by vnugent** [reply]
I'd definitely consider it for future projects.  Would you consider adding an FAQ explaining to those of us who are already familiar with Radix and want to adopt Daisy that you're solving 2 of our problems at once?
